widget/ui/src/components/Modal/Modal.tsx

```diff
@@ -106,17 +106,18 @@ export function Modal(props: PropsWithChildren<PropTypes>) {
                 </ModalHeader>
               )}
               <Content>{children}</Content>
-              {(hasLogo || footer) && (
-                <Footer>
-                  <div className="footer__content">{footer}</div>
-                  {hasLogo && (
-                    <div className="footer__logo">
-                      <Divider size={12} />
-                      <BottomLogo />
-                    </div>
-                  )}
-                </Footer>
-              )}
+
+              <Footer>
```

Comment:
In previous code, footer was not rendered when hasLogo || footer was false, but it seems it's different now. 🤔